src/kata6.joinNames.js

```diff
@@ -1,3 +1,8 @@
-const joinNames = (namesObj) => {};
-
+const joinNames = (namesObj) => {
+  const justNames = namesObj.map((people) => people.name);
+  const lastName = justNames.splice(-1);
+  const firstHalf = justNames.join(", ");
+  const lastStr = lastName.toString();
+  return `${firstHalf} & ${lastStr}`;
+};
```

Comment:
This works, but we can improve on it a bit.
First, the variable names. Calling the argument being passed in `namesObj` is misleading, as its an array, so this could be changed to just `people`
Then in the `map` function we could use a better placeholder name that reads more like english, eg person as person.name makes more sense than people.name what with it being a singular instance of people
We could also remove the toString method as its already a string, and condense it down a bit to remove an unnecessary variable. 
Lastly, its good practice to have a clear line before the export statement at the end of the file. Putting all that together we get:
```
const joinNames = (people) => {
  const names = people.map((person) => person.name);
  const finalName = names.splice(-1);
  return `${names.join(", ")} & ${finalName}`;
};

```